src/views/settings/Settings.sass

```diff
@@ -1,21 +1,35 @@
 .Settings
+    &[data-theme="dark"]
+        --background-color: #2f2f2f
+        --text-color: white
```

Comment:
So we change this to a media query that uses `prefers-color-scheme: dark` instead.

Reply:
I have done changes that you asked. Let me know if their is something else where I can do improvement in code. Thanks for the suggestion & review.

In new pull request I have done following changes:
1. add prefers-color-scheme media query for dark mode
2. refactor the previous code
3. Used @mixin to keep code dry.
4. Move the main `styles.css` in seperate folder `styles`, where add some new configs for sass

Thanks :)

Reply:
Sorry for taking a while to get back to you.

Yeah manual control is fine too, but the default should be "auto" which respects the OS scheme. So there would be 3 settings for theme: `light`, `dark`, and `auto`.

I am happy to make the adjustments if you like (especially given you've probably moved on from the PR a while ago :joy:).